test/EndToEndTests/Common/Microsoft.OData.Client.E2E.TestCommon/Samples/Tests/SampleTest.cs

```diff
@@ -0,0 +1,45 @@
+﻿//-----------------------------------------------------------------------------
+// <copyright file="SampleTest.cs" company=".NET Foundation">
+//      Copyright (c) .NET Foundation and Contributors. All rights reserved.
+//      See License.txt in the project root for license information.
+// </copyright>
+//------------------------------------------------------------------------------
+
+using Microsoft.AspNetCore.OData;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.OData.Client.E2E.TestCommon.Samples.Client;
+using Microsoft.OData.Client.E2E.TestCommon.Samples.Server;

```

Comment:
Do the client and server exist in the same project?

Reply:
The way have done it in this sample project is to have them in the same project but different folders. I don't see why they need to be in different projects.

Reply:
I think it's fine so long as we can easily tell the model classes apart, since they'll probably have the same names.